### PR TITLE
update: libGLU source URL: use https

### PR DESCRIPTION
**Pull request by asoete**
libGLU now has a https source available. The FTP source failed for me so
updating the easyconfigs to use https source instead of ftp.

closes gh-10624

### CODE REVIEW

File: easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-foss-2016a-Mesa-11.2.1.eb [hunk 1]
```diff
@@ -3,13 +3,13 @@ easyblock = 'ConfigureMake'
...
-homepage = 'ftp://ftp.freedesktop.org/pub/mesa/glu/'
+homepage = 'https://mesa.freedesktop.org/archive/glu/'
 description = """The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. """
 
 toolchain = {'name': 'foss', 'version': '2016a'}
 toolchainopts = {'pic': True}
 
-source_urls = ['ftp://ftp.freedesktop.org/pub/mesa/glu/']
+source_urls = ['https://mesa.freedesktop.org/archive/glu/']
 sources = ['glu-%(version)s.tar.bz2']
```
Comment (Micket) [inline easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-foss-2016a-Mesa-11.2.1.eb:14]:
```suggestion
checksums = ['1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12']

```

File: easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-foss-2016a.eb [hunk 2]
```diff
@@ -3,13 +3,13 @@ easyblock = 'ConfigureMake'
...
-homepage = 'ftp://ftp.freedesktop.org/pub/mesa/glu/'
+homepage = 'https://mesa.freedesktop.org/archive/glu/'
 description = """The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. """
 
 toolchain = {'name': 'foss', 'version': '2016a'}
 toolchainopts = {'pic': True}
 
-source_urls = ['ftp://ftp.freedesktop.org/pub/mesa/glu/']
+source_urls = ['https://mesa.freedesktop.org/archive/glu/']
 sources = ['glu-%(version)s.tar.bz2']
```
Comment (Micket) [inline easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-foss-2016a.eb:14]:
```suggestion
checksums = ['1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12']

```

File: easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-gimkl-2.11.5.eb [hunk 3]
```diff
@@ -3,13 +3,13 @@ easyblock = 'ConfigureMake'
...
-homepage = 'ftp://ftp.freedesktop.org/pub/mesa/glu/'
+homepage = 'https://mesa.freedesktop.org/archive/glu/'
 description = """The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. """
 
 toolchain = {'name': 'gimkl', 'version': '2.11.5'}
 toolchainopts = {'pic': True}
 
-source_urls = ['ftp://ftp.freedesktop.org/pub/mesa/glu/']
+source_urls = ['https://mesa.freedesktop.org/archive/glu/']
 sources = ['glu-%(version)s.tar.bz2']
```
Comment (Micket) [inline easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-gimkl-2.11.5.eb:14]:
```suggestion
checksums = ['1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12']

```

File: easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2016a-Mesa-11.2.1.eb [hunk 4]
```diff
@@ -3,13 +3,13 @@ easyblock = 'ConfigureMake'
...
-homepage = 'ftp://ftp.freedesktop.org/pub/mesa/glu/'
+homepage = 'https://mesa.freedesktop.org/archive/glu/'
 description = """The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. """
 
 toolchain = {'name': 'intel', 'version': '2016a'}
 toolchainopts = {'pic': True}
 
-source_urls = ['ftp://ftp.freedesktop.org/pub/mesa/glu/']
+source_urls = ['https://mesa.freedesktop.org/archive/glu/']
 sources = ['glu-%(version)s.tar.bz2']
```
Comment (Micket) [inline easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2016a-Mesa-11.2.1.eb:14]:
```suggestion
checksums = ['1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12']

```

File: easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2016a.eb [hunk 5]
```diff
@@ -3,13 +3,13 @@ easyblock = 'ConfigureMake'
...
-homepage = 'ftp://ftp.freedesktop.org/pub/mesa/glu/'
+homepage = 'https://mesa.freedesktop.org/archive/glu/'
 description = """The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. """
 
 toolchain = {'name': 'intel', 'version': '2016a'}
 toolchainopts = {'pic': True}
 
-source_urls = ['ftp://ftp.freedesktop.org/pub/mesa/glu/']
+source_urls = ['https://mesa.freedesktop.org/archive/glu/']
 sources = ['glu-%(version)s.tar.bz2']
```
Comment (Micket) [inline easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2016a.eb:14]:
```suggestion
checksums = ['1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12']

```

File: easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2016b.eb [hunk 6]
```diff
@@ -3,13 +3,13 @@ easyblock = 'ConfigureMake'
...
-homepage = 'ftp://ftp.freedesktop.org/pub/mesa/glu/'
+homepage = 'https://mesa.freedesktop.org/archive/glu/'
 description = """The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. """
 
 toolchain = {'name': 'intel', 'version': '2016b'}
 toolchainopts = {'pic': True}
 
-source_urls = ['ftp://ftp.freedesktop.org/pub/mesa/glu/']
+source_urls = ['https://mesa.freedesktop.org/archive/glu/']
 sources = ['glu-%(version)s.tar.bz2']
```
Comment (Micket) [inline easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2016b.eb:14]:
```suggestion
checksums = ['1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12']

```

File: easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2017a.eb [hunk 7]
```diff
@@ -3,13 +3,13 @@ easyblock = 'ConfigureMake'
...
-homepage = 'ftp://ftp.freedesktop.org/pub/mesa/glu/'
+homepage = 'https://mesa.freedesktop.org/archive/glu/'
 description = """The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. """
 
 toolchain = {'name': 'intel', 'version': '2017a'}
 toolchainopts = {'pic': True}
 
-source_urls = ['ftp://ftp.freedesktop.org/pub/mesa/glu/']
+source_urls = ['https://mesa.freedesktop.org/archive/glu/']
 sources = ['glu-%(version)s.tar.bz2']
```
Comment (Micket) [inline easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2017a.eb:14]:
```suggestion
checksums = ['1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12']

```

File: easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2017b.eb [hunk 8]
```diff
@@ -3,13 +3,13 @@ easyblock = 'ConfigureMake'
...
-homepage = 'ftp://ftp.freedesktop.org/pub/mesa/glu/'
+homepage = 'https://mesa.freedesktop.org/archive/glu/'
 description = """The OpenGL Utility Library (GLU) is a computer graphics library for OpenGL. """
 
 toolchain = {'name': 'intel', 'version': '2017b'}
 toolchainopts = {'pic': True}
 
-source_urls = ['ftp://ftp.freedesktop.org/pub/mesa/glu/']
+source_urls = ['https://mesa.freedesktop.org/archive/glu/']
 sources = ['glu-%(version)s.tar.bz2']
```
Comment (Micket) [inline easybuild/easyconfigs/l/libGLU/libGLU-9.0.0-intel-2017b.eb:14]:
```suggestion
checksums = ['1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12']

```

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 23 out of 23 (23 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.7.1908, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/cca3702f0d26f2d307169cffa28039e5 for a full test report.

**Comment by boegel** [comment]
close/re-open to wake up Travis, it seems like it was not informed of the last commit...

**Pull request merged**